cadl-tests/generate.sh

```diff
@@ -27,10 +27,12 @@ do
     generate $f
 done
 
-# run cadl ranch tests sources
-for f in $(find ./node_modules/@azure-tools/cadl-ranch-specs/http -name "*.cadl")
-do
-  if [[ $(realpath $f) == *"models"* || $(realpath $f) == *"hello"* ]]; then
-    generate $f
-  fi
-done
+## run cadl ranch tests sources
+#cp -rf node_modules/@azure-tools/cadl-ranch-specs/http .
+#for f in $(find ./http -name "*.cadl")
+#do
+#  if [[ $(realpath $f) == *"models"* ]]; then
```

Comment:
the `hello` no longer works in 0.34, as `getAllRoutes` return empty.

Reply:
OK, we can generate [srv-driven-2](https://github.com/Azure/cadl-ranch/tree/main/packages/cadl-ranch-specs/http/resiliency/srv-driven-2) code and test.

Reply:
Sure, if some of them works, you can add them back.